MoonAngleCondition/MoonAngleCondition.cs

```diff
@@ -161,6 +173,25 @@ public class MoonAngleCondition : SequenceCondition, IValidatable {
             return i.Count == 0;
         }
 
+        private double LorentzianSeparation(double distance, int width) {
+            // Moon-Avoidance Lorentzian formulated by the Berkeley Automated Imaging Telescope (BAIT) team
+            // Formula borrowed from ACP http://bobdenny.com/ar/RefDocs/HelpFiles/ACPScheduler81Help/Constraints.htm
+            var observerInfo = new ObserverInfo() {
+                Latitude = profileService.ActiveProfile.AstrometrySettings.Latitude,
+                Longitude = profileService.ActiveProfile.AstrometrySettings.Longitude,
+                Elevation = profileService.ActiveProfile.AstrometrySettings.Elevation,
+            };
+            var date = DateTime.UtcNow;
+            var jd = AstroUtil.GetJulianDate(date);
+            var moonPosition = Utility.Utility.GetMoonPosition(date, jd, observerInfo);
+            var moonage = moonPosition.RA / 29.53058770576;
```

Comment:
You can stick the 🪄 number in a `const` that has a suitable name

Reply:
Done